### ISSUE TITLE
Python seems to automatically delete the subprocess

### ISSUE THREAD

**Issue by srmnitc** [pyiron/pylammpsmpi#26]
When I set up a command like this-
```
from pylammpsmpi.lammps import LammpsLibrary
lmp = LammpsLibrary(cores=4)
lmp.units("metal")
lmp.boundary("p p p")
lmp.atom_style("atomic")
lmp.lattice("fcc", 4.0)
lmp.region("box", "block", 0, 10, 0, 10, 0, 30)
lmp.create_box(1, "box")
lmp.create_atoms(1, "box")
```
While executing you can see that there are 4 python threads started, but once it is done, the subprocess seems to end. A bit later any reference, such as `lmp.version` would throw a `BrokenPipe` error.

I will look into it a bit more.

**Comment by jan-janssen** [comment]
That is strange - this was working before, because that is exactly what we use in the pyiron interface. Or more over we send those as commands to the Lammps library https://github.com/pyiron/pyiron/blob/master/pyiron/lammps/interactive.py#L606 - Can you check if the same code works using commands?

**Comment by srmnitc** [comment]
will do.

**Comment by srmnitc** [comment]
I can confirm that the problem does not happen when I use `lmp.command`, I am going to raise separate issue to flag this and I will look into the problem.

**Comment by srmnitc** [comment]
Closing this now
